docs/notifications.json

```
@@ -28,5 +28,10 @@
     "id": 66,
     "title": "<b>MUI's company retreat in Tenerife</b> 🏝",
     "text": "In late June 2022, MUI team members gathered in the Canary Islands for a company retreat. <a style=\"color: inherit;\" data-ga-event-category=\"Blog\" data-ga-event-action=\"notification\" data-ga-event-label=\"2022-tenerife-retreat/\" href=\"/blog/2022-tenerife-retreat/\">Check out the recap here</a>."
+  },
+  {
+    "id": 67,
+    "title": "<b>Aggregate data like in Excel, but easier!</b> 🏝",
```

Probably wanna swap out the emoji here 😁 I like the abacus personally but maybe nobody under the age of 30 knows what that is.

```suggestion
    "title": "<b>Aggregate data like in Excel, but easier!</b> 🧮",
```